ipfix-flow-exporter/bpf_ipfix_egress.bpf.c

```diff
@@ -71,61 +71,80 @@ static u32 flow_key_hash (const flow_key_t f) {
     return hash_val;
 }
 
-static void update_flow_record(flow_record_t *flow_rec_from_map, flow_key_t flow_key,
-                               u16 pckt_size, u16 control_bit, u8 tos, u16 icmp_type,
-                               u8 ttl, u32 hash_key)
+struct update_flow_record_args {
```

Comment:
Can we move this struct definition to `bpf_ipfix_kern_common.h` file?

Reply:
done